telcell/data/models.py

```diff
@@ -41,6 +41,25 @@ def __len__(self) -> int:
         return len(self.measurements)
 
 
+class MeasurementPair:
+    """
+    A pair of two measurements. The pair can be made with different criteria,
+    for example the time difference between the two measurements. It always
+    contains the information from the two measurements it was created from.
+    """
+    def __init__(self, track_a, track_b):
+        self.track_a = track_a
+        self.track_b = track_b
+
+        self.time_difference = abs(track_a.timestamp - track_b.timestamp)
```

Comment:
Dit lijkt me logischer om als een `@property` te definieren die dynamisch berekend wordt i.p.v. eenmalig te berekenen in de `__init__()`.

Reply:
fixed!